content/docs/gke/deploy/monitor-iap-setup.md

```diff
@@ -209,28 +131,22 @@ problems:
     If you do not see a login prompt and you get a 404 error, the configuration
     of Cloud IAP is not yet complete. Keep retrying for up to 10 minutes.
 
-1. If you get an error `Error: redirect_uri_mismatch` after logging in, this 
-  means the list of OAuth authorized redirect URIs does not include your domain.
-
-    The full error message looks like the following example and includes the 
-    relevant links:
-
-    ```
-    The redirect URI in the request, https://mykubeflow.endpoints.myproject.cloud.goog/_gcp_gatekeeper/authenticate, does not match the ones authorized for the OAuth client. 
-    To update the authorized redirect URIs, visit: https://console.developers.google.com/apis/credentials/oauthclient/22222222222-7meeee7a9a76jvg54j0g2lv8lrsb4l8g.apps.googleusercontent.com?project=22222222222
-    ```
+1. If you get an error `Error: redirect_uri_mismatch` after logging in, this means the list of OAuth authorized redirect URIs does not include your domain.	
 
-    Follow the link in the error message to find the OAuth credential being used
-    and add the redirect URI listed in the error message to the list of 
-    authorized URIs. For more information, read the guide to 
-    [setting up OAuth for Cloud IAP](/docs/gke/deploy/oauth-setup/).
+    The full error message looks like the following example and includes the 	
+    relevant links:	
 
-## Expiry of the SSL certificate from Let's Encrypt
+    ```	
+    The redirect URI in the request, https://mykubeflow.endpoints.myproject.cloud.goog/_gcp_gatekeeper/authenticate, does not match the ones authorized for the OAuth client. 	
+    To update the authorized redirect URIs, visit: https://console.developers.google.com/apis/credentials/oauthclient/22222222222-7meeee7a9a76jvg54j0g2lv8lrsb4l8g.apps.googleusercontent.com?project=22222222222	
+    ```	
 
-Kubeflow runs an agent in your cluster to renew the Let's Encrypt certificate
-automatically. You don't need to take any action.
-For more information, see the [Let's Encrypt 
-documentation](https://letsencrypt.org/docs/integration-guide/).
+    Follow the link in the error message to find the OAuth credential being used	
+    and add the redirect URI listed in the error message to the list of 	
+    authorized URIs. For more information, read the guide to 	
+    [setting up OAuth for Cloud IAP](/docs/gke/deploy/oauth-setup/).	
 
-For questions and support about the certificate, visit 
-[Let's Encrypt support](https://community.letsencrypt.org/).
+## Next Steps
+* The Troubleshooting [Guide](https://www.kubeflow.org/docs/gke/troubleshooting-gke/) for Kubeflow.
```

Comment:
Suggested code, to avoid capital letters and use relative links in the doc site:

```
* The [GCP troubleshooting guide](/docs/gke/troubleshooting-gke/) for Kubeflow.
```